cpp/kiss_icp/core/Registration.cpp

```diff
@@ -87,19 +88,19 @@ std::tuple<Eigen::Vector3d, double> GetClosestNeighbor(const Eigen::Vector3d &po
     return std::make_tuple(closest_neighbor, closest_distance);
 }
 
-Associations FindAssociations(const std::vector<Eigen::Vector3d> &points,
-                              const kiss_icp::VoxelHashMap &voxel_map,
-                              double max_correspondance_distance) {
+Correspondences DataAssociation(const std::vector<Eigen::Vector3d> &points,
```

Comment:
I did some renaming that was pending from a previous PR, because we are now doing a massive renaming anyway I thought it was the right time. have a look

Reply:
Looks good to me!